apps/wfo-ui/next.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    reactStrictMode: false,
+    reactStrictMode: true,
```

Comment:
FYI. We had to disable this in the past because of EUI. Are all components verified that they still work?